internal/server/data/identity.go

```diff
@@ -64,7 +64,7 @@ func AssignIdentityToGroups(tx GormTxn, user *models.Identity, provider *models.
 		}
 		addIDs = append(addIDs, item)
 	}
-	if rows.Err() != nil {
+	if err := rows.Err(); err != nil {
```

Comment:
Looks like this would have returned nil previously, which doesnt seem intentional.